pkgs/applications/networking/p2p/qbittorrent/default.nix

```diff
@@ -1,44 +1,47 @@
-{ stdenv, fetchurl, pkgconfig, which
+{ stdenv, fetchFromGitHub, pkgconfig
 , boost, libtorrentRasterbar, qtbase, qttools, qtsvg
 , debugSupport ? false # Debugging
-, guiSupport ? true, dbus_libs ? null # GUI (disable to run headless)
+, guiSupport ? true, dbus ? null # GUI (disable to run headless)
 , webuiSupport ? true # WebUI
 }:
 
-assert guiSupport -> (dbus_libs != null);
+assert guiSupport -> (dbus != null);
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "qbittorrent-${version}";
-  version = "4.1.0";
+  version = "4.1.1";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/qbittorrent/${name}.tar.xz";
-    sha256 = "0fdr74sc31x421sb69vlgal1hxpccjxxk8hrrzz9f5bg4jv895pw";
+  src = fetchFromGitHub {
+    owner = "qbittorrent";
+    repo = "qbittorrent";
+    rev = "release-${version}";
+    sha256 = "09bf1jr2sfdps8cb154gjw7zhdcpsamhnfbgacdmkfyd7qgcbykf";
   };
 
-  nativeBuildInputs = [ pkgconfig which ];
+  # NOTE: 2018-05-31: CMake is working but it is not officially supported
+  nativeBuildInputs = [ pkgconfig ];
 
   buildInputs = [ boost libtorrentRasterbar qtbase qttools qtsvg ]
-    ++ optional guiSupport dbus_libs;
+    ++ optional guiSupport dbus; # D(esktop)-Bus depends on GUI support
 
-  # Otherwise qm_gen.pri assumes lrelease-qt5, which does not exist.
+  # HACK: 2018-05-31: Otherwise qm_gen.pri assumes lrelease-qt5, which does not exist.
```

Comment:
Actually this is not a hack, since this is how we are supposed to configure the path to lrelease: https://github.com/qbittorrent/qBittorrent/commit/f4dc5c6809f59b07ee4d10893dfba177357d27a5